docs/essentials/caching-dependencies-and-directories.md

```diff
@@ -118,6 +118,9 @@ with return code 0.
 In case of insufficient disk space, `cache store` frees disk space by deleting
 the oldest keys.
 
+**Note:** Data from a key is not overwritten by default when running `cache store`. 
```

Comment:
~~Active voice is preferred~~. Prefer active voice:

`cache store` does not overwrite data for an existing key.

you need to delete etc